src/components/sideBar/index.js

```diff
@@ -0,0 +1,21 @@
+import useTranslation from 'hooks/useTranslation';
+import React, { Component } from 'react';
+import logo from '../../assets/smilies.png';
+
+import './styles.scss';
+
+const SideBar = props => {
```

Comment:
WDYT if you destructure your components `props` just like this:
`const SidebAR = ({title, children}) =>`
in order to avoid writing `props.title`and `props.children`

Reply:
again, you are right 👌